src/App.js

```diff
@@ -21,15 +21,20 @@ const StyledApp = styled.div`
 
 function App() {
   const [cookies, setCookie, removeCookie] = useCookies(["sessionToken"]);
+  const cookieName = "sessionToken";
+  const cookieOptions = { path: "/" };
   const setToken = token => {
-    setCookie("sessionToken", token, { path: "/" });
+    setCookie(cookieName, token, cookieOptions);
```

Comment:
I'd be more explicit with the naming of the cookie, if someone else tries to add another cookie what would they call it?